### ISSUE TITLE
Document more ways to force a line break

### ISSUE THREAD

**Issue by mtsknn** [sembr/specification#6]
This might be too off-topic, but since there's already a "How do I force a line break?" entry in the FAQ, here are some more ways to force a line break.

#### AsciiDoc

`<br />` is **not** supported.

You can instead use a trailing space followed by a plus (i.e. ` +`).

Can be verified at <https://asciidoclive.com/edit/scratch/1>.

#### Haddock

Couldn't find anything online. I don't have Haskell installed so I can't test this myself.

#### Markdown

`<br />` and trailing spaces are supported as already mentioned in the FAQ entry.

Another way is to use a trailing backslash like this:

```md
lorem\
ipsum
```

This is supported at least by CommonMark (can be verified at <https://spec.commonmark.org/dingus/>) and GitHub Flavored Markdown (tested on GitHub).

IMO using a backslash is the cleanest way, though maybe not as obvious as `<br />`.

#### MediaWiki

`<br />` is supported as mentioned at <https://www.mediawiki.org/wiki/Help:Formatting#Paragraphs>.

#### Org mode

`<br />` is **not** supported (tested on GitHub). On the other hand, I don't know how well GitHub supports Org mode.

You can instead use two trailing backslashes (i.e. `\\`) as mentioned at <https://orgmode.org/manual/Paragraphs.html>.

#### reStructuredText

`<br />` is **not** supported (tested on GitHub). Same here that I'm not sure if it's a shortcoming in rST itself or in the rST parser/whatever used by GitHub.

Some methods are mentioned at <https://stackoverflow.com/q/51198270/1079869>. (None of them are very clean IMO.)

**Comment by mattt** [comment]
> This might be too off-topic, but since there's already a "How do I force a line break?" entry in the FAQ, here are some more ways to force a line break.

Not at all! Thanks so much for taking such a thorough look at the available options. 

I just opened #7, which incorporates this new information. Beyond any additions or corrections, I'll want to update the CSS to support nested definition lists ahead of merging it.
